pkg/cloudprovider/provider/azure/provider.go

```diff
@@ -156,6 +165,26 @@ var osPlans = map[providerconfigtypes.OperatingSystem]*compute.Plan{
 	},
 }
 
+var osDiskSKUs = []compute.StorageAccountTypes{
```

Comment:
Yeah sure, I didn't have anything to map the storage types to so I went with a slice, and then added the loop since I had a slice defined. A map would make it easier to check for existence. Let me change that.